src/platform_impl/ios/uikit/text_field.rs

```diff
@@ -0,0 +1,56 @@
+use super::UIView;
+use icrate::Foundation::{NSObject, NSString};
+use objc2::mutability::IsMainThreadOnly;
+use objc2::rc::Id;
+use objc2::runtime::{NSObjectProtocol, ProtocolObject};
+use objc2::{extern_class, extern_methods, extern_protocol, mutability, ClassType, ProtocolType};
+
+extern_class!(
+    #[derive(Debug, PartialEq, Eq, Hash)]
+    pub(crate) struct UITextView;
+
+    unsafe impl ClassType for UITextView {
+        #[inherits(NSObject)]
```

Comment:
Gives you access to more functionality on `UITextView`.

```suggestion
        #[inherits(UIResponder, NSObject)]
```